lib/login/login_view.dart

```diff
@@ -26,5 +27,94 @@ class LoginView extends View<LoginMessage, LoginModel, LoginUpdate> {
             child: const Icon(Icons.logout_rounded),
           ),
         ],
-      );
+      ));
 }
+
```

Comment:
pls remove this (only add ready and representative code in your pull request or add a todo with a notation when it will be done)